pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -863,6 +870,24 @@ func (oe *operationExecutor) generateUnmountDeviceFunc(
 				deviceToDetach.VolumeSpec.Name(),
 				unmountDeviceErr)
 		}
+		// Before logging that UnmountDevice succeeded and moving on,
+		// use mounter.DeviceOpened to check if the device is in use anywhere
+		// else on the system. Retry if it returns true.
+		deviceOpened, deviceOpenedErr := mounter.DeviceOpened(deviceToDetach.DevicePath)
```

Comment:
Maybe we should skip this check if `DevicePath` is empty? Once @jingxu97 makes her restart robustness change to unmount stray directories, her code may call UnmountDevice without a `DevicePath`. I'll defer to her.